yb-voyager/src/reporter/stats/streamImportReporter.go

```diff
@@ -55,53 +57,64 @@ func (s *StreamImportStatsReporter) Init(tdb tgtdb.TargetDB, migrationUUID uuid.
 	return nil
 }
 
+func (s *StreamImportStatsReporter) Finalize() {
+	s.refreshStats()
+	s.uitable.Stop()
+}
+
+var headerRow, seperator1, seperator2, seperator3, row1, row2, row3, row4, row5, row6, timerRow io.Writer
+
 func (s *StreamImportStatsReporter) ReportStats() {
 	displayTicker := time.NewTicker(10 * time.Second)
 	defer displayTicker.Stop()
-	table := uilive.New()
-	headerRow := table.Newline()
-	seperator1 := table.Newline()
-	seperator2 := table.Newline()
-	seperator3 := table.Newline()
-	row1 := table.Newline()
-	row2 := table.Newline()
-	row3 := table.Newline()
-	row4 := table.Newline()
-	row5 := table.Newline()
-	row6 := table.Newline()
-	timerRow := table.Newline()
-
-	table.Start()
+	s.uitable = uilive.New()
```

Comment:
call `defer s.finalize()` here instead of outside this.

Reply:
any specific reason? will it better than current.

Reply:
yeah but that's a parallel goroutine, how will it know that it needs to stop now? That info is known to streamChanges().